### DISCUSSION TITLE
a few questions

### DISCUSSION THREAD

**Discussion by webdsf** [mediacms-io/mediacms#61] [General]
hello, you can ask a few questions that have arisen.
1. When there are updates and what is already working on your site will be available, this is support for playing on mobile, as well as fixing problems with symbols.
2. Can you find out how to block the use of video on certain domains?
3. Since the adaptation is used, can you find out how to add a selection of multiple audio tracks in different languages?
4. How to delete automatically originals after transcode?
5. And you can find out where in the code you can change the profiles. I saw a separate file with the settings, are they?
I would need to increase crf. as it strongly presses and degrades the quality, and also so that the sound is copied without changes. Since I already coded it in aac.
A very necessary function, at least edit it in the code until a full release is released.

**Comment by Dan1ell** [comment]
As mentioned in point (1),  an update / feature release schedule would be pretty cool!

**Comment by mgogoulos** [comment]
hi,

> hello, you can ask a few questions that have arisen.
> 
> 1. When there are updates and what is already working on your site will be available, this is support for playing on mobile, as well as fixing problems with symbols.

we'll have to work on the milestones better to show what is currently being worked on.

> 2. Can you find out how to block the use of video on certain domains?

perhaps adding some block rules on nginx...

> 3. Since the adaptation is used, can you find out how to add a selection of multiple audio tracks in different languages?
not sure I understand this!

> 4. How to delete automatically originals after transcode?
it's not possible, could be added at some point to free disk space.Could be a configuration rule

> 5. And you can find out where in the code you can change the profiles. I saw a separate file with the settings, are they?
>    I would need to increase crf. as it strongly presses and degrades the quality, and also so that the sound is copied without changes. Since I already coded it in aac.
>    A very necessary function, at least edit it in the code until a full release is released.

files/helpers.py contains the related code, as you have spotted. Can give you more hints how commands for ffmpeg are generated (plan to document this)
